hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/EdgeLabelBuilder.java

```diff
@@ -149,6 +149,71 @@ public EdgeLabel create() {
         });
     }
 
+    private boolean hasSameProperties(EdgeLabel existedEdgeLabel) {
+        HugeGraph graph = this.graph();
+        Id sourceId = graph.vertexLabel(this.sourceLabel).id();
+        if (! existedEdgeLabel.sourceLabel().equals(sourceId)) {
+            return false;
+        }
+
+        Id targetId = graph.vertexLabel(this.targetLabel).id();
+
+        if (! existedEdgeLabel.targetLabel().equals(targetId)) {
+            return false;
+        }
+
+        if (this.frequency != existedEdgeLabel.frequency()) {
+            return false;
+        }
+
+        // this.enableLabelIndex == null, it means true.
+        if (this.enableLabelIndex == null || this.enableLabelIndex) {
+            if (! existedEdgeLabel.enableLabelIndex()) {
+                return false;
+            }
+        } else { // means false
+            if (existedEdgeLabel.enableLabelIndex() == false) {
```

Comment:
here this. enableLabelIndex = false, seems only `existedEdgeLabel.enableLabelIndex() == true` should return false.

Reply:
Good job, this is a bug.